### PR TITLE
Fix #15851 /wj without arg produce '\n'

### LINKED ISSUE #15851: Fix broken `/wj`
See https://ci.appveyor.com/project/radareorg/radare2/build/job/ibmo2yjs5580r7a7#L1907 for more output
```
[**]                  db\tools\rax2   OK  343 BR   17 XX    0 FX    1
[BR] /wj
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.t.check (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:874:14)
    at NewRegressions.checkTest (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:693:12)
    at NewRegressions.checkTestResult (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:729:33)
    at C:\projects\radare2\test\new\node_modules\node-r2r\index.js:145:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
[BR] /wj
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.t.check (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:874:14)
    at NewRegressions.checkTest (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:693:12)
    at NewRegressions.checkTestResult (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:729:33)
    at C:\projects\radare2\test\new\node_modules\node-r2r\index.js:145:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
[BR] /wj
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.t.check (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:874:14)
    at NewRegressions.checkTest (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:693:12)
    at NewRegressions.checkTestResult (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:729:33)
    at C:\projects\radare2\test\new\node_modules\node-r2r\index.js:145:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
[BR] /wj
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.t.check (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:874:14)
    at NewRegressions.checkTest (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:693:12)
    at NewRegressions.checkTestResult (C:\projects\radare2\test\new\node_modules\node-r2r\index.js:729:33)
    at C:\projects\radare2\test\new\node_modules\node-r2r\index.js:145:21
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

### PR DESCRIPTION
**Pull request by x0urc3**
- /wj without input did not return '[]'
- added string to test case

### CODE REVIEW

File: test/new/db/json/json1 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 /Oj
 /oj
 /Rj
-/wj BROKEN
+/wj asdf BROKEN
```
Comment (radare) [inline test/new/db/json/json1]: imho thats wrong and it should be showing a valid json in here too

Comment (radare) [reply test/new/db/json/json1]: even if no argument is passed

**Comment by x0urc3** [comment]
- Remove new line introduced by #3214 
- Tested /wj, /bj and /ij

**Comment by x0urc3** [comment]
- Remove new line introduced by #3214 
- Tested /wj, /bj and /ij

**Comment by radare** [comment]
Red

**Comment by XVilka** [comment]
Code Red

**Comment by x0urc3** [comment]
#15851 is really a good first (hard) issue. Will have a look again

**Comment by x0urc3** [comment]
@radare need some advice. I'm comparing the behaviour of '/j' and '/wj'. I believe they should behave consistently. Which one should I follow?

```
[0x000067d0]> /j   <- no space and no argument
[]
[0x000067d0]> /j   <- with space and no argument
Invalid keyword
                   <- newline here
[0x000067d0]> /wj   <- no space and no argument
                   <- newline here
[0x000067d0]> /wj    <- with space and no argument  
                   <- newline here
[0x000067d0]> 
```

**Comment by radare** [comment]
> On 25 Jan 2020, at 04:56, Khairulmizam Samsudin <notifications@github.com> wrote:
> 
> @radare <https://github.com/radare> need some advice. I'm comparing the behaviour of '/j' and '/wj'. I believe they should behave consistently. Which one should I follow?
> 
> [0x000067d0]> /j   <- no space and no argument
> []
> [0x000067d0]> /j   <- with space and no argument
> Invalid keyword
it's stderr
>                    <- newline here
> [0x000067d0]> /wj   <- no space and no argument
>                    <- newline here
> [0x000067d0]> /wj    <- with space and no argument  
>                    <- newline here
> [0x000067d0]> 
the output should be consistent, and no results must print [] (an empty array)

**Comment by radare** [comment]
ping

**Comment by x0urc3** [comment]
Found stray \n from strace output. Will trace using debugger
```
$> strace r2 -qc "/w" /bin/ls
------- cut here -------
rt_sigaction(SIGINT, {sa_handler=0x7f3132aed782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7f3133028540}, NULL, 8) = 0
stat("/bin/ls.r2", 0x7ffdb63e62d0)      = -1 ENOENT (No such file or directory)
stat("/usr/share/radare2/radare2rc", 0x7ffdb63e62d0) = -1 ENOENT (No such file or directory)
rt_sigaction(SIGINT, {sa_handler=0x7f3132aed782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7f3133028540}, NULL, 8) = 0
rt_sigaction(SIGINT, {sa_handler=0x7f3132aed782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7f3133028540}, NULL, 8) = 0
rt_sigprocmask(SIG_BLOCK, [WINCH], [], 8) = 0
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
------ cut here ----

$> strace r2 -qc "/wj" /bin/ls
------ cut here ----
rt_sigaction(SIGINT, {sa_handler=0x7fd12c699782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7fd12cbd4540}, NULL, 8) = 0
stat("/bin/ls.r2", 0x7ffd5ff36e70)      = -1 ENOENT (No such file or directory)
stat("/usr/share/radare2/radare2rc", 0x7ffd5ff36e70) = -1 ENOENT (No such file or directory)
rt_sigaction(SIGINT, {sa_handler=0x7fd12c699782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7fd12cbd4540}, NULL, 8) = 0
rt_sigaction(SIGINT, {sa_handler=0x7fd12c699782, sa_mask=[INT], sa_flags=SA_RESTORER, sa_restorer=0x7fd12cbd4540}, NULL, 8) = 0
write(1, "\n", 1)                       = 1
rt_sigprocmask(SIG_BLOCK, [WINCH], [], 8) = 0
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
------ cut here ----

**Comment by x0urc3** [comment]
$> r2 -qc "/wj " /bin/ls
Invalid keyword
$> r2 -qc "/wj" /bin/ls
Invalid keyword
$> r2 -qc "/wj asdf" /bin/ls
[]
$>

**Comment by radare** [comment]
Uhm. Are you using spaces instead of tabs?

**Comment by x0urc3** [comment]
My bad. New push used tabs

**Pull request merged**